### ISSUE TITLE
issue with saphyra.py

### ISSUE THREAD

**Issue by ReddyKousic** [H1R0GH057/Anonymous#37]
![image](https://user-images.githubusercontent.com/85285392/173321030-febdefcd-762a-4a53-b02e-12024ef23f11.png)
how to resolve it

**Comment by swordapi** [comment]
Use python2 saphyra.py http://target.com/

**Comment by Lamf4o** [comment]
Hey @ReddyKousic,

Here's the code for an HTTP/S Protocol.
Paste this at the end of the script :

```
                url = sys.argv[1]
		if url.count("/")==2:
			url = url + "/"
		m = re.search("http\://([^/]*)/?.*", url)
                m2 = re.search("https\://([^/]*)/?.*", url)
                try:
		    host = m.group(1)
                except:
                    host = m2.group(1)
		for i in range(700):
			t = HTTPThread()
			t.start()
		t = MonitorThread()
		t.start()
```

I hope we helped you, the script just needed some pretty simple editing.
- XForWorks

**Comment by iqbalwalker** [comment]
nice just missed it

**Comment by sinaj2889** [comment]
Use a http:// url you can just remove the s of a https site and it will still work

**Comment by swordapi** [comment]
@XFORWORKS I get a error

~/Anonymous $ python2 saphyra.py https://dstat.cc/
  File "saphyra.py", line 3535
    if url.count("/")==2:
    ^
IndentationError: unexpected indent

**Comment by swordapi** [comment]
@XFORWORKS 
Can you upload the by you exited script to mega or anonfiles?

**Comment by Lamf4o** [comment]
@sinaj2889
I could do that but the script is remade so it's not working like it was before. The error you have on that line is because of space input, see if you pasted it correctly. If there is a problem, tell me.

- XForWorks ( Retr0 )

**Comment by swordapi** [comment]
@XFORWORKS thanks for your fast reply the problem is if I just paste it will be all on one line and I dont think that will work and I tried doing the spaces but it didn't work

**Comment by swordapi** [comment]
@XFORWORKS can you add spaces or what do you mean by remade I can just replace it then by your version

**Comment by Lamf4o** [comment]
@sinaj2889
Here is an example of the space input. :
![image](https://user-images.githubusercontent.com/67760271/178070140-97eb91a3-752a-436e-907a-95c69453c1ce.jpeg)

- XForWorks ( Retr0 )

**Comment by swordapi** [comment]
@XFORWORKS What about pasting the whole script to paste.it

**Comment by XFORWORKS** [comment]
@sinaj2889
No. It's because of other Banner and it's remade in an other type of attack, it's more powerful than this script you have here publicly... if the script gets in the wrong place or person, it could lead to a lot of harm to companies. Just do the space input the way I send you.

- XForWorks ( Retr0 )

**Comment by swordapi** [comment]
@XFORWORKS okkk I'm working with my team against Russia so how can we contact you and what do you mean with false persons

**Comment by XFORWORKS** [comment]
@sinaj2889
Hmm? Okay, we'll talk on Discord about this.. Retr0#0001

- XForWorks ( Retr0 )

**Comment by swordapi** [comment]
@XFORWORKS this name is not valid on Discord

**Comment by XFORWORKS** [comment]
@sinaj2889
Missed a dot in it, sorry.
It's "Retr0.#0001"

- XForWorks ( Retr0 )

**Comment by mixc2** [comment]
‌
